pkg/smi/fake.go

```diff
@@ -83,6 +83,19 @@ func (f fakeMeshSpec) ListBackpressures() []*backpressure.Backpressure {
 	return f.backpressures
 }
 
+func (f fakeMeshSpec) GetBackpressurePolicy(svc service.MeshService) *backpressure.Backpressure {
+	for _, backpressure := range f.backpressures {
+		app, ok := backpressure.Labels["app"]
```

Comment:
We should open the `key` from the `key/value` pair of the label to be any string.

Reply:
This is a temporary hack to get backpressure working with the selector. I don't think the CRD should be using a label selector for the service in the first place, and fixing this would require updating the CRD and autogen code. @Jont828 are you comfortable updating the code?